src/Modules/Finance/Header/TargetSum/TargetSum.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+import styles from '../Form.module.css';
+
+interface ITargetSumProps {
+	value: string | number | any,
+	buttonValue: string,
+	clicked: any,

```

Comment:
Is it possible to make the type more specific than `any`?

Reply:
It's possible. And I'll do it later for sure.

Reply:
will be converted in a bug